rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1186,7 +1238,13 @@ private void adjustBalancesIfChangeOutputWasDust(BtcTransaction btcTx, Coin sent
         for (TransactionInput transactionInput : btcTx.getInputs()) {
             sumInputs = sumInputs.add(transactionInput.getValue());
         }
-        Coin change = btcTx.getOutput(1).getValue();
+
+        Coin change = Coin.ZERO;
+        try {
+            change = btcTx.getValueSentToMe(getActiveFederationWallet(false));
+        } catch (IOException e) {
+            logger.error("Unexpected error accessing storage while attempting to process release requests", e);
```

Comment:
I don't like the idea of catching this exception here. If there's an error accessing the storage to get the active federation wallet then `change` will simply be 0, that could result in unexpected behaviour

Reply:
Done